src/DGtal/geometry/surfaces/estimation/VoronoiCovarianceMeasureOnDigitalSurface.ih

```diff
@@ -127,7 +127,7 @@ VoronoiCovarianceMeasureOnDigitalSurface( ConstAlias< Surface > _surface,
         {
           Point p = *itPts;
           const EigenStructure& evcm = myPt2EigenStructure[ p ];
-          VectorN n = evcm.vectors.column( 2 );
+          VectorN n = evcm.vectors.column( Space::dimension-1 );
```

Comment:
The VCM normal corresponds to the greatest eigenvalue of the VCM covariance. If the eigenvectors are sorted in increasing eigenvalues, then your fix seems correct to me.